src/main/java/pl/poznan/put/jsontools/rest/JsonToolsController.java

```diff
@@ -0,0 +1,66 @@
+package pl.poznan.put.jsontools.rest;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import pl.poznan.put.jsontools.logic.*;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Validated
+@RestController
+@RequestMapping("/api")
+public class JsonToolsController {
+    private static final Logger logger = LoggerFactory.getLogger(JsonToolsController.class);
+
+    @RequestMapping(method = RequestMethod.GET, produces = "application/json")
+    public String get(@Valid @RequestBody JsonToolsFullRequest request) {
+        logger.debug("Got request:\n" + request.toString());
+
+        logger.info("Processing transforms.");
+
+        var transformer = new JsonTransformer();
+
+        for (var transform : request.transforms) {
+            switch (transform.name) {
+                case "remove-attributes":
+                    transformer.addTransform(new JsonTransformRemoveAttributes(transform.attributes));
+                    break;
+                case "retain-attributes":
+                    transformer.addTransform(new JsonTransformRetainAttributes(transform.attributes));
+                    break;
+                case "minify":
+                    transformer.addFinalTransform(new JsonTransformMinify());
+                    break;
+                case "format":
+                    transformer.addFinalTransform(new JsonTransformFormat());
+                default:
+                    break;
+            }
+        }
+
+        logger.info("Processing JSON data");
+        return request.data.parallelStream()
+                .map(transformer::execute)
+                .collect(Collectors.joining(","));
```

Comment:
This is not correct JSON.
If you want to see proper output in postman then this can be changed to:
```
return "[\n\t" + 
            request.data.parallelStream()
                .map(transformer::execute)
                .collect(Collectors.joining(",\n\t")) +
                "\n]";
```

Reply:
And don't forget to view response in 'raw' tab, not the 'pretty' one.

Reply:
> This is not correct JSON. If you want to see proper output in postman then this can be changed to:
> 
> ```
> return "[\n\t" + 
>             request.data.parallelStream()
>                 .map(transformer::execute)
>                 .collect(Collectors.joining(",\n\t")) +
>                 "\n]";
> ```

works fine, thanks